### PR TITLE
correct license wording in advisory board minutes

### PR DESCRIPTION
**Pull request by SethTisue**
Scala's license is absolutely standard BSD 3-clause, not merely
"BSD-style"

### CODE REVIEW

**Comment by SethTisue** [comment]
I will submit a separate PR in the scala-lang repo adding the "Scala is licensed under the BSD 3-Clause License " wording from https://github.com/scala/scala/blob/2.12.x/doc/LICENSE.md to the top of http://www.scala-lang.org/license.html

**Comment by BennyHill** [comment]
Is it important that the scala license has not always been "pure" BSD, it most definitely used to be a modified verion - see  https://github.com/scala/scala/commit/bf93057e2eb41629f69657ea4a60acac0b824fce ?

My main reason for highlighting this is to show that changing a license is not actually has hard as suggested in the minutes, so not an obstacle for migratation to apache 2.0, as @fommil suggested by proxy, and @milessabin in https://github.com/typelevel/scala#relationship-with-lightbend-scala

**Comment by SethTisue** [comment]
the previous change only standardized the wording without altering the meaning. it seems to me that success there has no bearing on what it would take to make a substantive change.

**Comment by fommil** [comment]
changing a BSD 3 Clause is trivial. EPFL agreeing to use Apache 2.0 going forward is the problem: they'd have to clear releases with their patent office.

**Comment by BennyHill** [comment]
ahh, OK. I was just curious, that's all. Thx :)

File: minutes/002-2016-q3.md [hunk 1]
```diff
@@ -321,8 +321,8 @@ sbt).  These topics could eventually become concrete proposals.
 
 #### The Scala license
 
-Bill summarized Sam's licensing concern as follows.  Scala has
-[its own BSD-style license](http://www.scala-lang.org/license.html),
+Bill summarized Sam's licensing concern as follows.  Scala uses
+[the BSD 3-clause license](http://www.scala-lang.org/license.html),
```
Comment (fommil) [inline minutes/002-2016-q3.md]: "a 3-Clause Simplified BSD License" if you want to be exact :wink: http://ensime.github.io/contributing/center/#licensing

**Pull request merged**